.eslintrc.json

```diff
@@ -1,3 +1,27 @@
 {
-    "extends": ["next/core-web-vitals", "plugin:react/jsx-runtime"]
+    "env": {
+        "jest": true
+    },
+    "globals": {
+        "it": true,
+        "JSX": true,
+        "expect": true,
+        "describe": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
+    "extends": [
+        "next/core-web-vitals",
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "prettier"
+    ],
+    "rules": {
+        "@typescript-eslint/no-unused-vars": [
+            "error",
+            { "ignoreRestSiblings": true }
+        ],
+        "react-hooks/exhaustive-deps": "off"
```

Comment:
are we sure about turning this one off? feels like it's bad practice to ignore these warnings unless there's a really good reason not to. in my experience, it can lead to some hard-to-diagnose bugs with stale values and it usually suggests that there's a shortcoming in the way the component code was designed. and, if the dev really needs to, they can just write an eslint disable comment, which has the added benefit of forcing them to consider their decision more concretely and sometimes prompting another comment to explain why. just my 2 cents

Reply:
not sure if it was even on previously or not but if we're making the eslintrc stricter i think it's a good idea to turn it on personally

Reply:
i can turn it back on, TBH the warning was only occurring within a couple hooks that I was unfamiliar with so I was hesitant about adding to the dependency array and it producing unwanted side effects. ill remove this and let the devs who are familiar with the implementation add to it as they come across it in the future